gips/atmosphere.py

```diff
@@ -505,13 +613,12 @@ def process_acolite(asset, aco_proc_dir, products):
         imgout = gippy.GeoImage(ofname, tmp, dtype, len(bands))
```

Comment:
What resolution do the Sentinel-2 products come out at?  

I ask because this is the one area which I thought there might be trouble, because the Sentinel-2 bands have different resolutions (and thus numbers of pixels).

Reply:
I looked at all the gdalinfo outputs for the sentinel2 products and they all have this line (which means 10m resolution I believe)?

`Pixel Size = (10.000000000000000,-10.000000000000000)`

Reply:
Assuming the projection is in meters (which it probably is) then yes that means pixels are 10x10 m.

Reply:
thanks bhbraswell, didn't know where to find the actual unit:
`    UNIT["metre",1, AUTHORITY["EPSG","9001"]],`

Reply:
OK.  So, I asked because Sentinel-2 bands have either 10, 20, or 60 meter resolution.  And I didn't know how ACOLITE was handling that.  A more certain answer would come from checking the resolution of each band as it is read out of the netCDF4 output file, but one can infer that ACOLITE is promoting everything to 10m resolution based on the code here, and that the visual inspection of the GIPS products (the ones mentioned in Tom's comment today) confirms the products are not distorted -- as a resolution mismatch would distort them.